tests/core/data_model/test_base.py

```diff
@@ -469,7 +469,7 @@ def test_get_field_message_type_valid_fields():
     ) as dm:
         # Non-message field
         thing_one = dm.ThingOne(1)
-        assert thing_one.get_field_message_type("foo") is None
+        assert thing_one.get_field_message_type("foo") == int
```

Comment:
Just curious. Does `is int` not pass? I'd expect that it would (and is a little tighter)